app/AppKernel.php

```diff
@@ -13,15 +13,25 @@
 class AppKernel extends Kernel
 {
     /**
-     * {@inheritdoc}
+     * Here are the bundles you want to add for your project
```

Comment:
I would have commented "Register the project custom bundles". The sentence coment inside the method is clear enough imho.